src/screens/KeyManagement/OCR2KeyCreate.test.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+import { MockedProvider } from '@apollo/client/testing'
+import {
+  OCR2KeysCreate,
+  Props as OCR2KeysCreateProps,
+} from 'screens/KeyManagement/OCR2KeysCreate'
+import { render, screen } from 'test-utils'
+import { OCR2_KEY_FAMILY } from 'hooks/queries/useOCR2KeysQuery'
+import userEvent from '@testing-library/user-event'
+const { queryByText } = screen
+
+function renderComponent(cardProps: OCR2KeysCreateProps, mocks: any[]) {
```

Comment:
```suggestion
function renderOCR2KeysCreateWithMocks(cardProps: OCR2KeysCreateProps, mocks: any[]) {
```

Reply:
Changed in [`6bb1ac3` (#38)](https://github.com/smartcontractkit/operator-ui/pull/38/commits/6bb1ac37ef5ef0b55deb0414fff2035857755311)